docs/pages/admin-guides/access-controls/sso/github-sso.mdx

```diff
@@ -348,12 +348,12 @@ You will receive the details of your user session within the CLI:
 
 </TabItem>
 
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem scope={["cloud"]} label="Teleport Enterprise">
 
 ```code
-> Profile URL:        https://mytenant.teleport.sh:443
+> Profile URL:        example.teleport.sh:443
```

Comment:
Same here. I suppose this issue is duplicated in a bunch of other places?

Reply:
![A705D1E8-8D9B-417C-9703-2DA08E816F77_4_5005_c](https://github.com/user-attachments/assets/d45b5054-a021-4ed1-a8cb-db8d746321b3)

I updated the 16 entries that included 'https' in this URL, while the majority of the entries were already using the version without 'https'. I will revert those 16.